core/src/main/java/hudson/model/View.java

```diff
@@ -1000,6 +1001,15 @@ public synchronized void doDoDelete(StaplerRequest req, StaplerResponse rsp) thr
      */
     public abstract Item doCreateItem( StaplerRequest req, StaplerResponse rsp ) throws IOException, ServletException;
 
+    /**
+     * A new API method to get the allowed {$link TopLevelItem}s and its categories.
```

Comment:
This description is going to look strange in a very short while (when it is no longer new). Just use `@since` on the method.